roles/nougat/tasks/dependencies.yml

```diff
@@ -16,6 +16,45 @@
     chdir: "{{ frc_align_dest }}/build"
     creates: "{{ frc_align_dest }}/LICENSE"  
 
+- name: Fetch KmerGenie
+  get_url:
+    url: "{{ kmer_source }}/{{ kmer_version }}.tar.gz"
+    dest: "/scratch/{{ kmer_version }}.tar.gz"
+
+- name: Unpack KmerGenie
+  unarchive:
+    src: "/scratch/{{ kmer_version }}.tar.gz"
+    dest: "/scratch/"
+    copy: no
+    mode: g+rwxs
+
+- name: Create KmerGenie installation directory
+  command: mv "/scratch/{{ kmer_version }}" "{{ kmer_dest }}"
+  args:
+    creates: "{{ kmer_dest }}"
+    removes: "/scratch/{{ kmer_version }}"
+
+- name: Install Kmergenie
+  shell: "make"
+  args:
+    chdir: "{{ kmer_dest }}"
+
+- name: Fix KmerGenie permission issue
+  file:
+    path: "{{ kmer_dest }}/kmergenie"
+    state: touch
+    mode: "u+rwxs"
```

Comment:
You probably don't want to set the binary setuid, as that means the binary will be run with the permission belonging to the file owner (i.e. the user that deployed the sw).